eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxow.hs

```diff
@@ -192,9 +189,8 @@ instance
       MissingRequiredDatums x y -> Sum MissingRequiredDatums 2 !> To x !> To y
       NotAllowedSupplementalDatums x y -> Sum NotAllowedSupplementalDatums 3 !> To x !> To y
       PPViewHashesDontMatch m -> Sum PPViewHashesDontMatch 4 !> To m
-      MissingRequiredSigners x -> Sum MissingRequiredSigners 5 !> To x
-      UnspendableUTxONoDatumHash x -> Sum UnspendableUTxONoDatumHash 6 !> To x
-      ExtraRedeemers x -> Sum ExtraRedeemers 7 !> To x
+      UnspendableUTxONoDatumHash x -> Sum UnspendableUTxONoDatumHash 5 !> To x
+      ExtraRedeemers x -> Sum ExtraRedeemers 6 !> To x
```

Comment:
You shouldn't change the serialization of other constructors. This creates an unnecessary breakage for the tooling. Considering that this serialization is not used in Conway, it would not be the end of the world, but it is good practice. The only time when we can adjust tags like this is for an era that has never been deployed on mainnet yet.
```suggestion
      UnspendableUTxONoDatumHash x -> Sum UnspendableUTxONoDatumHash 6 !> To x
      ExtraRedeemers x -> Sum ExtraRedeemers 7 !> To x
```